.chloggen/migrate-internal-aws-to-sdk-v2.yaml

```diff
@@ -0,0 +1,29 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: internal/aws
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Upgrading internal/aws to adopt AWS SDK v2
```

Comment:
please remove the changelog, as this is invisible to users and no changes to the internal API are published outside this repository.

Reply:
ok thank u sir